CMSgov/cms-mdct-qmr
# MDCT-1830: CCW-AD Deviations from Measure Specs (Bug)

## Purpose

For a specific measure (CCW-AD), if a user completed an NDR set and selected deviations, QMR would not correctly validate whether or not those deviations were filled. 

To test:

- `./dev local`
- Navigate to **Adult Core Set Measures** -> **CCW-AD**
- **Are you reporting on this measure?** -> _Yes_
- **What is the status of the data being reported?** -> _I am reporting final data_
- **Measurement Specification** -> _HHS_
- **Data Source** -> _Administrative Data_

... I promise, we'll get there.

⚠️ Fill at least one (1) NDR set ⚠️ 
- **Deviations from Measure Specifications** -> _Yes_
- Select one of the specifications, then **Numerator || Denominator || Other**
- DON'T fill in the **Explain** field
- Scroll down to **Validate Measure** and click

You should see this wonderful error:

<img width="1260" alt="Screen Shot 2022-08-26 at 8 20 45 AM" src="https://user-images.githubusercontent.com/99458559/186902195-82d22735-aeaa-438e-8d35-570ae70dde20.png">

Now, let's make it ✨ go away ✨ 

- Go back up to the deviations section and type in anything, truly anything, whatever you want, maybe write a haiku for me
- Scroll down and click on **Validate Measure**

Bet you don't see that wonderful error anymore, huh? 🧙🏽 And you know what? It'll be the same for FY2021 _and_ FY2022.

Alright, now if you don't believe me, try all of the steps above with any other measure. Go ahead, do it. You know you want to.

#### Linked Issues to Close

Closes [MDCT-1830](https://qmacbis.atlassian.net/browse/MDCT-1830).

## Approach

It's kind of hack-y, if I'm being real with you. But it's an edge case.

#### Pull Request Creator Checklist


#### Pull Request Reviewer/Assignee Checklist
